parser.js

```diff
@@ -274,7 +274,7 @@
     function createClassRange(min, max, from, to) {
       // See 15.10.2.15:
       if (min.codePoint > max.codePoint) {
-        throw SyntaxError('invalid range in character class');
+        bail('invalid range in character class: ' + min.raw + '-' + max.raw, from, to);
```

Comment:
nit: The output from this message looks like this

```
invalid range in character class: a-- at position 1\n    [a--z]\n     ^
```

The colon separates the message - how do you feel about using quotes and removing the colon instead - like:

```
invalid range in character class "a--" at position 1\n    [a--z]\n     ^
```

?